util.py

```diff
@@ -383,3 +383,24 @@ def update_nested_dict(dict1: Dict[str, Any], dict2: Dict[str, Any]):
         else:
             dict1[k] = v
     return dict1
+
+
+def parse_text_dict(path: Union[str, tk.Path]) -> Dict[str, str]:
+    """
+    Loads the text dict at :param:`path` making sure not to trigger line counter overflow.
+    """
+
+    with uopen(path, "rt") as text_dict_file:
+        txt = text_dict_file.read()
+
+    # remove leading and trailing dict brackets
+    txt = txt.strip().strip("{}").strip()
```

Comment:
This is too sloppy for me. You expect a very certain structure. Do that more explicitly, checking for that:
```suggestion
    # remove leading and trailing dict brackets
    txt = txt.strip()
    assert txt[:1] == "{" and txt[-1:] == "}"
    txt = txt[1:-1]
```